.github/CONTRIBUTING.md

```diff
@@ -15,6 +15,14 @@ This can be automated by setting up the pre-commit hook properly.
 
     ln -s ../../scripts/prepare-commit.sh .git/hooks/pre-commit
 
+Windows equivalent for a symlink using PowerShell in elevated priviledges mode from the .git\hooks folder. This will create hardlink which can be used from git-bash shell.
```

Comment:
I'm by no means a Windows expert, so sorry if the question is trivial.
Why is there a requirement for elevated privileges to do something in a user folder?

(Sidenote, there is a typo in privileges)

Reply:
@whatnick Why the powershell reference?  I'd use cmd too (if cygwin is unavailable).
@m-kuhn on windows links are privileged business.